src/Sarif/Core/PropertyBagHolder.cs

```diff
@@ -19,6 +19,12 @@ namespace Microsoft.CodeAnalysis.Sarif
     /// </summary>
     public class PropertyBagHolder : IPropertyBagHolder
```

Comment:
>PropertyBagHolder [](http://example.com/codeflow?start=17&length=17)

This class is a base type for all our core OM. This means that any utilization of an OM type will invoke this static cctor.